docopt_macros failing to compile on rustc 1.16.0-nightly (4ecc85beb 2016-12-28)

Running into a error when compiling the `docopt_macros` crate. I am honestly not to sure on how to fix the issue, but happy to work on a fix if someone can point me in the right direction.

**compiler error message**  
  ```
  Compiling docopt_macros v0.6.87
  error[E0560]: struct `syntax::ast::Path` has no field named `global`
  --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/docopt_macros-0.6.87/src/macro.rs:321:9
      |
  321 |         global: false,
      |         ^^^^^^^ `syntax::ast::Path` does not have this field

  error[E0308]: mismatched types
  --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/docopt_macros-0.6.87/src/macro.rs:324:25
      |
  324 |             parameters: ast::PathParameters::AngleBracketed(tystr),
      |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `std::option::Option`, found enum `syntax::ast::PathParameters`
      |
      = note: expected type `std::option::Option<syntax::ptr::P<syntax::ast::PathParameters>>`
      = note:    found type `syntax::ast::PathParameters`

  error: aborting due to 2 previous errors
  error: Could not compile `docopt_macros`.
  ```

--- a/docopt_macros/src/macro.rs
+++ b/docopt_macros/src/macro.rs
@@ -318,10 +318,9 @@ fn ty_vec_string(cx: &ExtCtxt) -> P<ast::Ty> {
     };
     cx.ty_path(ast::Path {
         span: sp,
-        global: false,
         segments: vec![ast::PathSegment {
             identifier: ident("Vec"),
-            parameters: ast::PathParameters::AngleBracketed(tystr),
+            parameters: Some(P(ast::PathParameters::AngleBracketed(tystr))),
         }]
     })
 }

